### ISSUE TITLE
Document choosing different loss functions

### ISSUE THREAD

**Issue by stsievert** [adriangb/scikeras#219]
As discussed in #218, it's hard to choose the loss automatically (categorical cross entropy loss is best for independent labels, binary cross entropy is okay for multiple labels).

Motivation for filing this issue: https://github.com/adriangb/scikeras/pull/218#discussion_r589842418
